### ISSUE TITLE
Allow users to click anywhere within a point to select it

### ISSUE THREAD

**Issue by rhamilton03** [openforis/collect-earth-online#956]
Some users have reported frustration with selecting individual or sets of individual points when interpreting a plot. Currently, the user has to click on the perimeter of the point to select it. It would be much easier for the interpreters to be able to click anywhere within the point to select it.

**Comment by sirmspencer** [comment]
Can you please confirm this is recent?  With last big release I have added code that allows users to click inside the sample points.  I also made them bigger to allow them to be easier to click on.

**Comment by sirmspencer** [comment]
The issue seemed to be an outdated package just on collect.earth.  Please let me know if this come back.

**Comment by rhamilton03** [comment]
Hi Matt,

I have a group in Costa Rica that just started a new interpretation project
last week. They are the ones that brought it to my attention. The project
was created last week. I also tried it last week on an existing project and
found the same thing. However, I tested it this morning and found I could
click anywhere inside the plot and it selected the point.

Thanks for looking into it.

Regards,

Randy Hamilton
